packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -437,8 +438,25 @@ export async function getBookings({
         booking.attendees = booking.attendees.filter((attendee) => attendee.email === user.email);
       }
 
+      let previousBooking = null;
+
+      if (booking.fromReschedule) {
+        const rescheduledBooking = await prisma.booking.findUnique({
+          where: {
+            uid: booking.fromReschedule,
+          },
+          select: {
+            rescheduledBy: true,
+          },
+        });
+        if (rescheduledBooking) {
+          previousBooking = rescheduledBooking;
```

Comment:
all you wanna return is "rescheduledBy" 

I don't think you should put it in an object called previousBooking